File: web/src/data/pro-radio-pricing.ts
/**
 * Pro-Radio subscription pricing.
 * Keep in sync with backend/src/pro-radio-subscription/pro-radio-subscription.constants.ts
 * and mobile/lib/core/constants/pro_radio_pricing.dart.
 */

export const PRO_RADIO_REGULAR_CENTS = 999;
export const PRO_RADIO_INTRO_CENTS = 499;

export function formatProRadioPriceUsd(cents: number): string {
  return `$${(cents / 100).toFixed(2)}`;
}

export const PRO_RADIO_REGULAR_DISPLAY = formatProRadioPriceUsd(
  PRO_RADIO_REGULAR_CENTS,
);
export const PRO_RADIO_INTRO_DISPLAY = formatProRadioPriceUsd(
  PRO_RADIO_INTRO_CENTS,
);

File: mobile/lib/core/constants/pro_radio_pricing.dart
/// Pro-Radio subscription pricing.
/// Mirrors web/src/data/pro-radio-pricing.ts and
/// backend/src/pro-radio-subscription/pro-radio-subscription.constants.ts.

const int proRadioRegularCents = 999;
const int proRadioIntroCents = 499;

String formatProRadioPriceUsd(int cents) =>
    '\$${(cents / 100).toStringAsFixed(2)}';

final String proRadioRegularDisplay =
    formatProRadioPriceUsd(proRadioRegularCents);
final String proRadioIntroDisplay = formatProRadioPriceUsd(proRadioIntroCents);
